### PR TITLE
Refactor Managed Apps dashboard after UX review

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/roadmap/issues/322

This PR

- Aligns Managed Apps Usage and Managed Apps Upgrades dashboards for consistency and with the UX guidelines. https://intranet.giantswarm.io/docs/product/ux/dashboards/

### Checklist

- [ ] ~~Update changelog in CHANGELOG.md in an end-user friendly language.~~

These 2 dashboards are merged but not released yet. I'd like to get a review from SIG UX before doing that.

### CODE REVIEW

File: helm/dashboards/dashboards/shared/managed-apps-upgrades.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
         }
       ],
       "title": "App installations with an upgrade available",
-      "type": "bargauge"
+      "type": "stat"
```
Comment (rossf7) [inline helm/dashboards/dashboards/shared/managed-apps-upgrades.json:103]: Using stats for both dashboards for consistency.

File: helm/dashboards/dashboards/shared/managed-apps-usage.json [hunk 2]
```diff
@@ -71,20 +71,20 @@
...
+      "id": 3,
       "options": {
         "colorMode": "value",
         "graphMode": "none",
         "justifyMode": "auto",
         "orientation": "auto",
         "reduceOptions": {
           "calcs": [
-            "lastNotNull"
+            "last"
```
Comment (rossf7) [inline helm/dashboards/dashboards/shared/managed-apps-usage.json:87]: Using last for all the calculations.

File: helm/dashboards/dashboards/shared/managed-apps-usage.json [hunk 3]
```diff
@@ -261,12 +261,12 @@
           },
           {
             "selected": false,
-            "text": "giantswarm",
+            "text": "Giant Swarm Catalog",
```
Comment (rossf7) [inline helm/dashboards/dashboards/shared/managed-apps-usage.json:264]: Aligning the text with Happa.

**Comment by rossf7** [comment]
Usage dashboard shows all managed apps in the installation.

Screen Shot 2021-09-15 at 6 05 28 PM

Upgrades dashboard shows all managed apps with an upgrade available.

Screen Shot 2021-09-15 at 6 05 38 PM

**Comment by rossf7** [comment]
Managed Apps Usage dashboard shows all managed apps for the installation.

Screen Shot 2021-09-16 at 9 50 09 AM

Managed Apps Upgrades dashboard shows all managed apps with an upgrade.

Screen Shot 2021-09-16 at 9 50 01 AM

File: helm/dashboards/dashboards/shared/managed-apps-upgrades.json [hunk 4]
```diff
@@ -205,7 +209,7 @@
               },
               "App Count": {
                 "aggregations": [
-                  "max"
+                  "last"
```
Comment (rossf7) [inline helm/dashboards/dashboards/shared/managed-apps-upgrades.json:212]: Using last for all the calculations.

**Review by piontec — APPROVED** [review]
Nice!

**Comment by marians** [comment]
All customer facing dashboards should be linked in this panel:

image

The two here are missing. Atlas can probably help you with adding them.

**Review by MarcelMue — APPROVED** [review]
Adding in the overview can be a follow up.

**Comment by rossf7** [comment]
> The two here are missing. Atlas can probably help you with adding them.

Thanks @marians I'll check with Atlas how to do that 👍

**Comment by rossf7** [comment]
@marians Links are added. PTAL

**Comment by marians** [comment]
Please also use sentence case where title case is used currently. Thanks!

**Comment by rossf7** [comment]
@marians Changes made. PTAL :)

File: helm/dashboards/values.yaml [hunk 5]
```diff
@@ -12,6 +12,14 @@ home:
...
     description: "find out which alerts are currently active for an installation, and see which are overruled by silences or inhibitions"
     provider: "shared"
+  - name: "Managed Apps Upgrades"
+    link: "/d/6LDk8YVnk/managed-apps-upgrades"
+    description: "find out which managed apps have an upgrade available"
+    provider: "shared"
+  - name: "Managed Apps Usage"
+    link: "/d/O74c1gI7k/managed-apps-usage"
+    description: "find out which managed apps are installed for an installation"
+    provider: "shared"
```
Comment (marians) [inline helm/dashboards/values.yaml:22]:
```suggestion
  - name: "Managed apps upgrades"
    link: "/d/6LDk8YVnk/managed-apps-upgrades"
    description: "find out which managed apps have an upgrade available"
    provider: "shared"
  - name: "Managed apps usage"
    link: "/d/O74c1gI7k/managed-apps-usage"
    description: "find out which managed apps are installed for an installation"
    provider: "shared"
```

**Comment by marians** [comment]
Please also adjust the dashboard titles to sentence case

**Comment by rossf7** [comment]
Please also adjust the dashboard titles to sentence case

@marians My bad I missed those. Done.

**Review by marians — APPROVED** [review]
Ross and I talked about few tweaks still to make. With that LGTM.

**Comment by rossf7** [comment]
@marians Thanks for the great feedback <3

Here is the updated version.

Screen Shot 2021-09-17 at 10 51 50 AM

**Pull request merged**